contrib/libcpuid-cmake/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-if (NOT ARCH_ARM)
+if (NOT ARCH_ARM AND NOT ARCH_PPC64LE)
```

Comment:
cpuid is x86 only, let's replace

NOT ARCH_ARM AND NOT ARCH_PPC64LE to ARCH_AMD64.